drivers/scheduler/k8s/specs/elasticsearch/px-elasticdata.yaml

```diff
@@ -1,3 +1,11 @@
+kind: StorageClass
```

Comment:
Keep `StorageClass` is it's separate file since that belongs to storage. While keeping in one file works, it's good to keep storage and app in separate files.

Do the the same for other specs.

Reply:
@harsh-px  But if we have them separate, does torpedo guarantee that PVC is created before the application?

Reply:
Yes, Schedule implementation in k8s does that. It performs storage operations before app/core operations.